### DISCUSSION TITLE
Insights global filtering

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#8710] [Feature Requests]
An idea for Directus insights:
Being able to select a filter for all elements in the dashboard.
Example usecase:
 - Make a dashboard with all time registered by employees
 - Have one global filter to filter them for a project
 - This way you can see the total and you can see the amount of time spent for the selected project.
- You can use the variable inside filters, this way you can make dynamic views.

**Comment by benhaynes** [comment]
This is difficult to set for _any_ filter, since panels can pull data from different collections with different fields. One thing that we can set globally is the date/time range for Time-Series. That way you could adjust the time range globally, and all panels could update to that (regardless of collection, field).

**Reply by Jeyakumarrathnaselvi** [reply]
Thanks, Ben. We need the feature when the user needed when a common field available for all collection data sets of all blocks. For example: even if a common field does not exist in all collections when we set such a filter then add the date range field part of each block then make a call. it may return empty if a particular field does not exist in a few collections but it doesn't create any problem, right?

without a filter, insight a dashboard is static and it doesn't deliver insight it supposed to deliver

**Reply by benhaynes** [reply]
👍  — and just to be clear, each panel can be filtered. What you're looking for is an easier way to adjust filters globally from the dashboard level. I agree that this would be quite powerful... it's just lower on the feature request board, and so a PR or sponsorship would be useful in getting this done more quickly.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
